django_app/redbox_app/templates/accessibility-statement.html

```diff
@@ -1,46 +1,39 @@
-{% import "macros.html" as macros %}
-{% extends "base_generic_gov.html" %}
+{% extends "base.html" %}
 
-<!--TODO: Update system url and other variables-->
-
-{% block title %}
-  <title>Accessibility statement - Redbox Copilot - GOV.UK</title>
-{% endblock %}
+{% set pageTitle = "Accessibility statement" %}
```

Comment:
The old style is the GDS way of doing it, might be worth keeping it that way

Reply:
Interesting. I'm definitely keen to follow existing convention where possible, but the "new" approach offers the following advantages:
* Reduces the risk of a mistake (e.g. someone makes a typo on one page)
* It doesn't need to be a block as it's only plain text that changes
* Reduces repetition and therefore makes updating the site name easier
* Reduces the risk of inconsistencies creeping in

It's just a question of whether these advantages are enough to move away from the GDS way. The advantages are only small. What do you reckon?
(if we change back I'll create a ticket, as it'll mean changes on other pages too - still a tiny job though)